### ISSUE TITLE
Self host

### ISSUE THREAD

**Issue by chronoss09** [RealYoti/project-amber#4]
Hi

Can i use it without internet on selfhosting ?

**Comment by Yoti** [comment]
Yes, you can. Only VitaShell tied to Internet (that's why it has "download" word in the menu).

**Comment by chronoss09** [comment]
I test and i have a blackscreen after the exploit and my psvita shutdown :(

**Comment by Yoti** [comment]
Are you sure that you moved payload.bin file after building with SDK? From payload dir to the upper (root) one.

**Comment by Darthagnon** [comment]
Does Project Amber not require downloading Henkaku, gamesd.skprx, etc. externally? 

The similar Henlo JB downloads Henkaku, VitaDeploy, gamesd.skprx, etc. externally, seems impossible to self-host without recompiling every time IP changes: https://github.com/SKGleba/henlo_jb/issues/8

I'm wondering if there's a way to self-host the additional tools like VitaDeploy and VitaShell, or even, say, an arbitrary vpk...

**Comment by Darthagnon** [comment]
Compiled payload.bin can be downloaded from http://vitawiki.xyz/henlo/payload.bin, in case anyone else is looking at self-hosting.

**Comment by chronoss09** [comment]
It's only for Henkaku or for Vitadeploy too ?
